source/Stargate-Model/CrossOriginResourceSharingRequestHandlerBuilder.class.st

```diff
@@ -0,0 +1,11 @@
+Class {
+	#name : #CrossOriginResourceSharingRequestHandlerBuilder,
+	#superclass : #Object,
+	#category : #'Stargate-Model-CORS'
+}
+
+{ #category : #building }
+CrossOriginResourceSharingRequestHandlerBuilder >> build [
+
+	self subclassResponsibility 
```

Comment:
We are using the idea of returning the subclass responsibility when the implementation is expected to return an object other than self.
So in this case I *think* it should be `^ self subclassResponsibility '